ui/.eslintrc

```diff
@@ -1,7 +1,10 @@
 {
     "extends": ["react-app","standard"],
+    "plugins": ["react-hooks"],
```

Comment:
Should we also be adding the Patternfly React plugin?

Reply:
separate PR for that. I didn't want to get bogged down with the patternfly eslint stuff